packages/flutter_tools/lib/src/base/features.dart

```diff
@@ -0,0 +1,245 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
```

Comment:
I'm not sure lib/src/base/ is the right place for this file. I think lib/src is probably more appropriate.

Reply:
Moved